src/server/controllers/match.controller.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const matchSchema = require('mongoose').model('match');
+import mailer from 'nodemailer';
+
+/*
+ * Methods about mentoring request, accept or reject including E-mail Service
```

Comment:
In the future, we may need to refactor out the email engine in a separate file so we can use email functions in different part of code. :)